modules/checkout/config/schema/commerce_checkout.schema.yml

```diff
@@ -70,6 +70,9 @@ commerce_checkout.commerce_checkout_pane.login:
 commerce_checkout.commerce_checkout_pane.order_summary:
   type: commerce_checkout_pane_configuration
   mapping:
+    use_views:
```

Comment:
How would the UI look, then? I added this because then the UI will use `#states` to toggle the dropdown. 

Should the select list be "Display option:" and "Use template" or underneath is a random list of views? UI feels weird that way.